app/Http/Class/PaypalPayment.php

```diff
@@ -0,0 +1,56 @@
+<?php 
+
+namespace App\Http\Class;
+
+use Srmklive\PayPal\Services\PayPal as PayPalClient;
+use App\Http\Interface\PaymentInterface;
+use App\Models\Cart;
+use Illuminate\Support\Facades\Redirect;
+
+class PaypalPayment implements PaymentInterface {
+    
+    public static function handle($data){
+
+        $payment = new PaypalPayment;
+
+        return $payment->process($data);
+    }
+
+    protected function process($data){
+
+        $provider = new PayPalClient;
+        $provider->setApiCredentials(config('paypal'));
+        $paypalToken = $provider->getAccessToken();
+        $response = $provider->createOrder([
+            "intent" => "CAPTURE",
+            "application_context" => [
+                "return_url" => route('cart.success'),
+                "cancel_url" => route('cart.cancel'),
+            ],
+            "purchase_units" => [
+                0 => [
+                    "amount" => [
+                        "currency_code" => "PHP",
+                        "value" => $data['total'],
+                    ]
+                ]
+            ]
+        ]);
```

Comment:
Little things that can be done to refactor this function.

For example, the array being sent as parameter to the `createOrder` could have been done by a different function.

```php
$response = $provider->createOrder(
  $this->buildOrderPayload($data['total'])
);

...

protected function buildOrderPayload($total)
{
  return [
    'intent' => 'CAPTURE',
    ...
    'purchase_units' => [
      ...
      'amount' => [
        ...
        'value' => $total
      ]
    ]
  ];
}
```